vue-dash/src/components/MyListingsSubpage.vue

```diff
@@ -1,17 +1,19 @@
 <template>
-    <div class="subpage">
-        <h2>I'm the MyListings Subpage</h2>
```

Comment:
Can we changes spaces from 4 -> 2? I also like 4 spaces but the Google HTML/CSS styleguide asks for 2
https://google.github.io/styleguide/htmlcssguide.html

Reply:
Will do!